2.0.0 changes

cs-abhishek-shukla opened the pull request:
- Details in release_notes.md
- Updated contents.md with new playbooks and marked the old PBs as deprecated
- Updated the setup.md with configuration instructions for setting up dev env

cs-abhishek-shukla committed:
2.0.0 changes
- Details in release_notes.md
- Updated contents.md with new playbooks and marked the old PBs as deprecated
- Updated the setup.md with configuration instructions for setting up dev env

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -0,0 +1,51 @@
+
+- Removed the dependency on the **Attachments** records to create export templates
+
+    - Added *Content and Settings Export Templates* in export template record
```
i think extra space got added?

cs-abhishek-shukla replied:
It is for this effect -- the sub-bulleting.

![image](https://user-images.githubusercontent.com/100339208/230374168-687b1c60-9eec-4b31-8563-c741afe682c4.png)

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -0,0 +1,51 @@
+
+- Added Pull Request (PR) title as an issue comment in *Create Pull Request* playbook
+
+- Pass input record to child playbook in The *Fetch Latest Changes* playbook
```
I guess, this is a internal working, do we really need to mention this point here? @cs-sujeet-ahirrao  please confirm

cs-abhishek-shukla replied:
Users must know what is happening even if it behind the scenes. However, since it is not user-facing I have not documented it in the SP doc elsewhere.

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -0,0 +1,51 @@
+
+- Pass input record to child playbook in The *Fetch Latest Changes* playbook
+
+- Renamed *Get User Confirmation* step to *Confirm Apply Latest Changes* in *Pull Latest Content from GitHub* playbook
```
Same as above. Here we can mentioned ux change, if there is any due to this change

cs-abhishek-shukla replied:
Users must know what is happening even if it behind the scenes. However, since it is not user-facing I have not documented it in the SP doc elsewhere.

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -0,0 +1,51 @@
+
+- Renamed *Get User Confirmation* step to *Confirm Apply Latest Changes* in *Pull Latest Content from GitHub* playbook
+
+- Removed redundant *Update Import Job* and *Modify Configuration Option* steps from *Setup Development Environment* playbook
```
Can we just say, Enhanced and optimised *Setup Development Environment* playbook?

cs-abhishek-shukla replied:
A detailed release note establishes trust and gives an impression of transparency, IMO.

cs-sujeet-ahirrao committed:
Update release_notes.md

Updated release notes

cs-sujeet-ahirrao committed:
Update contents.md

Updated Solution pack contents

cs-abhishek-shukla committed:
2.0.0 changes
- Add info that user has to accept the inviattion for collaborator
- Need to run the configure CICD wiz again after upgrade of SP
- before upgrading the SP, upgrade the GIT connector manually
- Post upgrade, delete  newly added export templates
- GitHub v1.1.0 dependency
- Upgrade instructions
- Codeload service URL usage
- Deactivate PBs during upgrade
